bskyweb/templates/base.html

```diff
@@ -315,6 +315,133 @@
   <meta property="og:site_name" content="Bluesky Social" />
   <link type="application/activity+json" href="" />
 
+  <script>
+    try {
+      const InterRegular = new FontFace(
```

Comment:
would it be sufficient to use `<link>` preload tags?

Reply:
Discussed in Slack, I think we're gonna give preload a go in #5431